analysis/topeft_run2/analysis_processor.py

```diff
@@ -322,7 +314,7 @@ def process(self, events):
         e_fo = ele[ele.isPres & ele.isLooseE & ele.isFO]
 
         # Attach the lepton SFs to the electron and muons collections
-        AttachElectronSF(e_fo, year=year, looseWP=ele_pres_id_tag) #Run3 ready
+        AttachElectronSF(e_fo, year=year, looseWP="none" if is_run3 else "wpLnoiso") #Run3 ready
```

Comment:
```suggestion
        AttachElectronSF(e_fo, year=year, looseWP=None if is_run3 else "wpLnoiso") #Run3 ready
```

Reply:
I'm confused by this change: shouldn't this force the code to crash in run3 ( https://github.com/TopEFT/topeft/pull/450/files#diff-bad2815db1cdd28a50951857cadd189aad367df4d391bd9fc43a6d7187715018L1067  ) ?

Reply:
Yeah, sorry. Ignore this comment.